R/wb_styles.R

```diff
@@ -873,12 +877,12 @@ set_cell_style <- function(wb, sheet, cell, value) {
 create_dxfs_style <- function(
     font_name      = "Calibri",
     font_size      = "11",
-    font_color     = c(rgb = "FF9C0006"),
+    font_color     = wb_color(rgb = "FF9C0006"),
     numFmt         = NULL,
     border         = NULL,
-    border_color   = getOption("openxlsx2.borderColour", "black"),
+    border_color   = wb_color(getOption("openxlsx2.borderColour", "black")),
```

Comment:
_color_ not _colour_?  We may want to check for the _u_ spelling anywhere else.

Reply:
Yeah, I'll create a search & replace PR for that, not gonna write deprecate functions for all these changes.